File: ws/morillo/u2/ws24 Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Eagle.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author David Morillo, JavaSquad, DCCO-ESPE
 */
public class Eagle extends Bird{

    public Eagle(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    @Override
    public void Feed(int food) {
        System.out.println("Feeding the Eagle with " + food + " squirrel");
    }
}

File: hw/bonifaz/u2/HW18ZooInPythonJavaScript/zoojs/ec.edu.espe.zoo.model/Eagle.js
const { Bird } = require('./Bird.js'); // Importar clase Bird desde otro archivo

class Eagle extends Bird {
    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    Feed(food) {
        throw new Error("Not supported yet.");
    }
}

module.exports = { Eagle };
